shell/go.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
nit: Use `/usr/bin/env bash` over `/bin/bash` to ensure we're using the user's bash version

Reply:
```suggestion
#!/usr/bin/env bash
```